saskatoon/member/views.py

```diff
@@ -113,3 +117,16 @@ def get_context_data(self, **kwargs):
         context['title'] = _("Organization Update")
         context['cancel_url'] = reverse_lazy('beneficiary-list')
         return context
+
+
+class PasswordChangeView(auth_views.PasswordChangeView):
+    template_name = 'registration/change_password.html'
+    form_class = PasswordChangeForm
+
+    def get_context_data(self, **kwargs):
+        context = super(PasswordChangeView, self).get_context_data(**kwargs)
+        return context
```

Comment:
What is this doing? looks like you don't need it

Reply:
true! removing...